.github/workflows/build_test_and_deploy_next.yml

```diff
@@ -0,0 +1,192 @@
+name: Build Test & Deploy next
+
+on:
+  push:
+     branches:
+       - master
+env:
+  NPM_REGISTRY: "https://registry.npmjs.org/"
+
+jobs:
+     Build_prerequisites:
+      runs-on: ubuntu-latest
+      steps:
+        - name: Git clone repository
+          uses: actions/checkout@v2
+
+        - name: Cache Node modules
+          uses: actions/cache@v2
+          with:
+            path: '**/node_modules'
+            key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
+
+        - name: Build icon-library
+          run: |
+            yarn install
+            yarn --cwd packages/icon-library build
+
+        - name: Build design-tokens
+          run: yarn --cwd packages/design-tokens build
+
+        - name: Create tar
+          run: |
+            cp -R packages/icon-library/dist distil
+            cp -R packages/design-tokens/dist distdt
+            tar -czvf dist.tar.gz distil distdt
+
+        - name: Persist icon-library
```

Comment:
Maybe we should rename this to persist artefacts or something?

Reply:
Fair point as its now doing more than just the icon-library, I'll rename.

Reply:
Renamed step, also did the same for build_and_test_feature.